src/tiledb/cloud/bioimg/ingestion.py

```diff
@@ -47,20 +46,34 @@ def ingest_tiff_udf(
 
         :param io_uris: Pairs of tiff input - output tdb uris
         :param config: dict configuration to pass on tiledb.VFS
-        :param workers: Number of threads that will spawn for parallelizing ingestion
         """
 
+        from tiledb import filter
         from tiledb.bioimg.converters.ome_tiff import OMETiffConverter
 
+        compressor = kwargs.get("compressor", None)
+        if compressor:
+            compressor_name = compressor.get("_name", None)
+            if compressor_name:
+                compressor_args = {
+                    k: None if not v else v
+                    for k, v in compressor.get("attrs", {}).items()
+                }
```

Comment:
to match the below, this would be

```python
# Copy to not mutate the passed-in dict
compressor_args = dict(compressor)
compressor_name = compressor_args.pop("_name")
```
and use the existing code for the rest.